source/mainnet/net/resources/release-notes-mainnet.rst

```diff
@@ -9,8 +9,16 @@ Release notes
    :backlinks: none
 
 
+
 Mainnet 3: Alpha Centauri 3.0
 ==============================
+January 13, 2022
+
+Concordium Desktop Wallet v1.3.1
+--------------------------------
+-   Fixed issue that caused the wallet to crash when inspecting identities with missing date attributes.
```

Comment:
Does this same note need to be added to the same release notes for testnet in PR 387?